webapp/packages/core-app/src/NavigationTree/NavigationTreeNode/NavigationNode/NavigationNodeControl.tsx

```diff
@@ -38,8 +50,12 @@ export const NavigationNodeControl: React.FC<Props> = observer(function Navigati
   node,
 }) {
   const context = useContext(TreeNodeContext);
+  const connectionId = NodeManagerUtils.connectionNodeIdToConnectionId(node.id);
+  const { connectionInfo } = useConnectionInfo(connectionId);
+  const disconnected = connectionInfo ? !connectionInfo.connected : false;
```

Comment:
better to declare `let disconnected = false`
also use `useService(ConnectionInfoResource)` instead `useConnectionInfo`
and 
```
  if(node.objectFeatures.includes(EObjectFeature.dataSource)) {
    const connectionInfo = connectionInfoResource.get(NodeManagerUtils.connectionNodeIdToConnectionId(node.id));
    disconnected = !connectionInfo?.connected;
  }
```